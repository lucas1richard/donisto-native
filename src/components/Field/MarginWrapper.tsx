import * as React from 'react';
import { View, StyleSheet } from 'react-native';
import theme from 'theme/iftheme';

export interface IMarginWrapperProps {
  children: any;
}

const MarginWrapper = ({ children }: IMarginWrapperProps) => (
  <View style={styles.wrapper}>
    {children}
  </View>
);

export default MarginWrapper;

const styles = StyleSheet.create({
  wrapper: {
    marginBottom: theme.fs / 2
  }
});
