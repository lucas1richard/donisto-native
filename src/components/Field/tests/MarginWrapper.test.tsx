import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MarginWrapper, { IMarginWrapperProps } from '../MarginWrapper';

describe('<MarginWrapper />', () => {
  const renderComponent = (props: IMarginWrapperProps) => toJson(shallow(<MarginWrapper {...props} />));
  it('renders with just text', () => {
    expect(renderComponent({ children: 'mockText' })).toMatchSnapshot();
  });
});