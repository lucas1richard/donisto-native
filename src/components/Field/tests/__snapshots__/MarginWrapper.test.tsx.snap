// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarginWrapper /> renders with just text 1`] = `
<Component
  style={
    Object {
      "marginBottom": 10,
    }
  }
>
  mockText
</Component>
`;
